docbot/metrics.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+from prometheus_client import Histogram
+
+
+@dataclass
+class Metrics:
+    github_latency = Histogram(
+        name='github',
+        documentation='Time spent processing from Github service'
```

Comment:
1. Processing what?
2. Github -> GitHub

Reply:
Fixed. Metric of processing a request, thanks!